content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -725,7 +725,7 @@ jobs:
 
 #### Example using action inside a different private repository than the workflow
 
-Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). (You will need to replace `PERSONAL_ACCESS_TOKEN` with the name of the secret you created.)
+Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). You will need to replace `PERSONAL_ACCESS_TOKEN` in the example below with the name of the secret you created.
```

Comment:
suggest replacing organisation with organization for spelling consistency in our docs

Reply:
Also - do we need to confirm the fact that the PAT will have the permissions of the user creating it? Eg may need a service account.

Reply:
```suggestion
Your workflow must checkout the private repository and reference the action locally. You will need to generate a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token) and add it to your [repository](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository
Duplicate of #) or [organisation](/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization). You will need to replace `PERSONAL_ACCESS_TOKEN` in the example below with the name of the secret you created.
```

Reply:
@Ice991 while those two links point to the same page, they are for two different sections on that page, one for an organisation, another is for a repository